### PR TITLE
Gaussian distribution fix

### PR DESCRIPTION
**Pull request by RemDelaporteMathurin**
We noticed that the gaussian distribution function was not normalised therefore its area was not unity!

### CODE REVIEW

**Review by kaelyndunnell — APPROVED** [review]

File: src/hisp/festim_models/mb_model.py [hunk 1]
```diff
@@ -317,8 +318,9 @@ def make_B_mb_model(
 
     vertices = np.concatenate(  # 1D mesh with extra refinement
         [
-            np.linspace(0, 30e-9, num=200),
-            np.linspace(30e-9, L, num=200),
+            np.linspace(0, 5e-9, num=300),
```
Comment (RemDelaporteMathurin) [inline src/hisp/festim_models/mb_model.py:321]: 300 vertices in the first 5 nm seems a bit overkill imo

File: src/hisp/festim_models/mb_model.py [hunk 2]
```diff
@@ -538,8 +540,8 @@ def make_B_mb_model(
 
     ############# Settings #############
     my_model.settings = F.Settings(
-        atol=1e-15,
-        rtol=1e-15,
+        atol=1e10,
+        rtol=1e-5,
```
Comment (RemDelaporteMathurin) [inline src/hisp/festim_models/mb_model.py:544]: can we try 1e-10 for `rtol`?

File: src/hisp/helpers.py [hunk 3]
```diff
@@ -40,7 +40,9 @@ def update(self, t: float):
 def gaussian_distribution(
     x: npt.NDArray, mean: float, width: float, mod=ufl
 ) -> ufl.core.expr.Expr:
-    return mod.exp(-((x[0] - mean) ** 2) / (2 * width**2))
+    return mod.exp(-((x[0] - mean) ** 2) / (2 * width**2)) / (
+        np.sqrt(2 * np.pi * width**2)
+    )
```
Comment (RemDelaporteMathurin) [inline src/hisp/helpers.py:45]: We should take this opportunity to document `gaussian_distribution`

File: src/hisp/plamsa_data_handling/main.py [hunk 4]
```diff
@@ -223,15 +223,15 @@ def get_heat(self, pulse: Pulse, bin: SubBin | DivBin, t_rel: float) -> float:
             raise ValueError(f"Invalid pulse type {pulse.pulse_type}")
 
         if pulse.pulse_type == "FP":
-            heat_total = data["heat_total"][bin_index]
+            heat_total = data["heat_total"][bin_index]+0.11e+06 # photon radiation heat adjustment
```
Comment (RemDelaporteMathurin) [inline src/hisp/plamsa_data_handling/main.py:226]: magic number alert!

```suggestion
            photon_heat_radiation = 0.11e6  # W/m2
            heat_total = data["heat_total"][bin_index]+ photon_heat_radiation 
```

File: src/hisp/plamsa_data_handling/main.py [hunk 5]
```diff
@@ -223,15 +223,15 @@ def get_heat(self, pulse: Pulse, bin: SubBin | DivBin, t_rel: float) -> float:
...
             heat_ion = data["heat_ion"][bin_index]
             if isinstance(bin, SubBin):
                 heat_val = heat_total - heat_ion * (1 - bin.wetted_frac)
             else:
                 heat_val = heat_total
         elif pulse.pulse_type == "RISP":
             if isinstance(bin, SubBin):
-                heat_total = data["heat_total"]
+                heat_total = data["heat_total"]+0.11e+06 # photon radiation heat adjustment
```
Comment (RemDelaporteMathurin) [inline src/hisp/plamsa_data_handling/main.py:234]: 🪄 Magic number alert

```suggestion
                photon_radiation_heat = 0.11e6  # W/m2
                heat_total = data["heat_total"]+photon_radiation_heat 
```

File: src/hisp/festim_models/mb_model.py (hunk 2 above)
Comment (kaelyndunnell) [reply src/hisp/festim_models/mb_model.py:544]: was getting really bad results with 1e-10 -- i can try something between?

Comment (RemDelaporteMathurin) [reply src/hisp/festim_models/mb_model.py:544]: I don't see why this would give bad results ... let's keep it like that and make a note of it

Comment (kaelyndunnell) [reply src/hisp/festim_models/mb_model.py]: lowest i can get it with good results is 1e-8

**Pull request merged**